source/content/symlinks-assumed-write-access.md

```diff
@@ -149,13 +149,44 @@ You can also verify success using `dir`:
 
 </Accordion>
 
+<Accordion title="Nitropack" id="nitropack-panel" icon="wrench">
+
+As discussed in [WordPress Plugins and Themes with Known Issues](/plugins-known-issues), [Nitropack](https://wordpress.org/plugins/nitropack/) assumes write access to the `wp-content/nitropack` folder and to `advanced.cache.php`.
+
+<Alert title="Note" type="info">
+
+Manually create the target folders `code/wp-content/uploads/nitropack` and `code/wp-content/uploads/advanced-cache.php` for Dev, Test, and Live environments.
+
+</Alert>
+
+1. In the command line, navigate to `code/wp-content/uploads` in your Dev environment. Or, if you are using an SFTP client (such as [FileZilla](https://filezilla-project.org/)), navigate to `files/`.
+
+1. Create a `nitropack` folder and an `advanced-cache.php` file. Be sure to delete any existing `advanced-cache.php` that is present in the `./uploads` directory before creating the file:
+
+  ```bash{promptUser: user}
+  mkdir ./nitropack​ && touch ./advanced-cache.php
+  ```
+
+1. Repeat steps 1 and 2 for your Test and Live environments.
+
+1. Create a symlink in the `code/wp-content` directory in your Dev environment:
+
+  ```bash{promptUser: user}
+  ln -s ./uploads/nitropack/ ./nitropack
+  ln -s ./uploads/advanced-cache.php ./advanced-cache.php
+  ```
+
+1. Commit changes to the Live environment.
```

Comment:
should the changes be committed directly to Live? or are we telling users to sync the content from the Multidev or Dev [through Test and to Live](https://pantheon.io/docs/pantheon-workflow)?

Reply:
@michellecolon-pantheon , this PR is otherwise good to go - I just want to confirm that we're telling users the right process in this step (cc @orlandoayeras )

Reply:
@EdwardAngert absolutely, you are right! They need to commit that symlink code changes from the Dev or multidev they are working on and push it to the Test and Live (tho they can actually do it directly to Live but it is much riskier as this path is more of a hotfix). Thanks for working on this PR!

Reply:
@orlandoayeras and @EdwardAngert - I've updated, if you want to take a look at the latest!